azure/table_azure_express_route_circuit.go

```diff
@@ -0,0 +1,230 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-07-01/network"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAzureExpressRouteCircuit(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_expressroute_circuit",
```

Comment:
@ParthaI Is the tableName correct, since it doesn't matches the fileName?
As per terraform, table name should be `azure_express_route_circuit`

Reply:
Updated the table name.